README.md

```diff
@@ -76,45 +223,59 @@ You can also right-click on any element in the ARM Template Outline view to init
   - Variable iteration ("copy blocks"), see Azure [documentation](https://docs.microsoft.com/en-us/azure/azure-resource-manager/resource-group-create-multiple#variable-iteration)
   - Sort template and template sections alphabetically
 
-## Extension configurations
+## Extension configuration
```

Comment:
I think these would be much better in a short table format - these take up too much space for relatively minor features and make the readme look more complex and confusing IMHO.

Reply:
Fixed.